cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -613,6 +619,7 @@ static DataprocConf create(Map<String, String> properties) {
     long pollCreateJitter = getLong(properties, "pollCreateJitter", 20);
     long pollDeleteDelay = getLong(properties, "pollDeleteDelay", 30);
     long pollInterval = getLong(properties, "pollInterval", 2);
+    long jobStatusPollInterval = getLong(properties, "jobStatusPollInterval", 2);
```

Comment:
Don't have a strong opinion here, but it may be better to re-use the existing pollInterval for the static provisioner instead of introducing a new one. Unless there's some reason to keep them separate?

Reply:
Addressed. 
Main reason was to be able to change this polling frequency separately. But agree that it may not matter much.